trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,232 @@
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={JupyterIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Jupyter <br/> Compatible
+                        </h1>
+                        <p>
+                            Mito is compatible with Jupyter Hub, Jupyter Lab, and Jupyter Notebooks, so you don’t need to manage any new infrastructure.  
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={PrivateIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Completely <br/> Private
+                        </h1>
+                        <p>
+                            With the Mito Pro and Enterprise plans, not a single message is sent back to Mito. Mito is designed from the ground up with security at its core. 
+                        </p>
+                    </div>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={DocsIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Practical <br/> Documentation
+                        </h1>
+                        <p>
+                            Great documentation is a must even for the most intutive tools. Mito’s docs are easily consumable, so your users can get started without any training.  
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={SupportIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Personal <br/> Support
+                        </h1>
+                        <p>
+                            The founders of Mito are around to help train, debug, or ideate on new features 24/7. 
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            <section>
+                <div className={pageStyles.subsection + ' ' + pageStyles.subsection_column + ' center'}>
+                    <h1>
+                        Make your team Python Self-Sufficient
```

Comment:
I love the idea of this section - the difference in a workflow before and after Mito. My only major comment on this page is that the design of this section makes it pretty hard to read and parse. It felt like it took me about 5 seconds to understand what I was looking at - and Jake left a similar comment on the mockup about it just being a wall of text.

I think the best thing to design this section as is pretty much a calendar week (as that what it is) - which is just a really standard way we're used to looking at weekdays!

 I took a really quick shot at it in mockups here: https://www.figma.com/file/C1ATxXNsIIKkj9Lb0ZRQOI/trymito.io?node-id=1064%3A341

Let me know your thoughts!